src/it/java/org/jboss/tools/intellij/kubernetes/BasicTests.java

```diff
@@ -142,6 +142,7 @@ public static void closeGotItPopup() {
 
     public static void closeOpenedEditors() {
         List<SingleHeighLabelFixture> singleHeighLabelsList = robot.findAll(SingleHeighLabelFixture.class, byXpath("//div[@class='SingleHeightLabel']"));
+        System.out.println("Next opened editors will be closed: " + singleHeighLabelsList);
```

Comment:
wouldnt it be better to log instead of sys-out? a log could be set to be printed to std out but also to a file?

Reply:
fixed